composer.json

```diff
@@ -60,7 +60,8 @@
     },
     "autoload-dev": {
         "psr-4": {
-            "Barryvdh\\LaravelIdeHelper\\Tests\\": "tests"
+            "Barryvdh\\LaravelIdeHelper\\Tests\\": "tests",
+            "Database\\Factories\\": "tests/Console/ModelsCommand/Factories/Factories"
```

Comment:
What is the reason for this?

I suggest to avoid such extra legwork and just use the correct namespace for the file in `tests/Console/ModelsCommand/Factories/Factories/SimpleFactory.php` and adjust it where used.